Autopilot/PathManager/Src/trackFinder.cpp

```diff
@@ -0,0 +1,17 @@
+#include <math.h>
+#include "trackFinder.hpp"
+
+constexpr double DEG2RAD {3.14159265358979323/180};
+
+double getTrack(double latitudeSpeed, double longitudeSpeed)
+{   
+    double trackInRad = atan2(longitudeSpeed,latitudeSpeed); //the x and y are flipped so that 0 rad is north
+    if(trackInRad < 0)
+    {
+        return (trackInRad/DEG2RAD + 360);
```

Comment:
division takes much longer than multiplication in general. constexpr are evaluated at compile time, so I'd say you should make it a factor that you multiply rather than one you divide. It's possible that the compiler may optimize this but I'm not sure. May as well make this a habit.